features/picture.yml

```diff
@@ -1,9 +1,9 @@
-draft_date: 2024-06-05
 name: <picture>
-description: The `<picture>` element TODO.
+description: The `<picture>` element is used for responsive images.
 spec: https://html.spec.whatwg.org/multipage/embedded-content.html#the-picture-element
 group: html
 caniuse: picture
 compat_features:
   - api.HTMLPictureElement
   - html.elements.picture
+  - html.elements.source
```

Comment:
The `<source>` element was first introduced for `<video>`, so it should either be part of that feature, or possibly its own feature.

Reply:
Split out into a separate element